src/lib/isMD5.js

```diff
@@ -0,0 +1,10 @@
+import assertString from './util/assertString';
+
+/* eslint-disable no-control-regex */
```

Comment:
I wasn't sure really so I followed the existing src with regex (lib/isAscii.js) which also has lint suppression.